### PR TITLE
README.md revision

### PR DESCRIPTION
**Pull request by marcospb19**
Trying to use Markdown styling to highlight nice stuff here

Commit on this pull-request branch to adapt this to your taste, if you would like to merge it

Thanks

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,50 +1,46 @@
...
+**Note:** If you don't use the provided installer for your platform, make sure that you add `CMake`'s bin folder to your path.
 
-2. Navigate into the source directory, and type:
+#### 1. Navigate into the source directory, create build folder and run `CMake`:
 
-```
+```sh
 mkdir build
 cd build
+cmake [-G generator] [-DBUILD_SHARED_LIBS=on|OFF] ..
```
Comment (jbeder) [inline README.md]: This is now YAML_BUILD_SHARED_LIBS

File: README.md [hunk 2]
```diff
@@ -1,50 +1,46 @@
...
+## How to Build
 
-1. Download and install [CMake](http://www.cmake.org) (Resources -> Download).
+`yaml-cpp` uses [CMake](http://www.cmake.org) to support cross-platform building. Install [CMake](http://www.cmake.org) _(Resources -> Download)_ before proceeding. The basic steps to build are:
 
-**Note:** If you don't use the provided installer for your platform, make sure that you add CMake's bin folder to your path.
+**Note:** If you don't use the provided installer for your platform, make sure that you add `CMake`'s bin folder to your path.
 
-2. Navigate into the source directory, and type:
+#### 1. Navigate into the source directory, create build folder and run `CMake`:
```
Comment (jbeder) [inline README.md]: Why the four-# header for this?

File: README.md [hunk 3]
```diff
@@ -1,50 +1,46 @@
...
+    * On OS X, you might use "Xcode".
+    * On a UNIX-like system, omit the option (for a Makefile).
 
-  * yaml-cpp defaults to building a static library, but you may build a shared library by specifying `-DYAML_BUILD_SHARED_LIBS=ON`.
+  * `yaml-cpp` builds a static library by default, you may want to build a shared library by specifying `-DBUILD_SHARED_LIBS=ON`.
 
   * For more options on customizing the build, see the [CMakeLists.txt](https://github.com/jbeder/yaml-cpp/blob/master/CMakeLists.txt) file.
 
-4. Build it!
+#### 2. Build it!
```
Comment (jbeder) [inline README.md]: ... and here.

File: README.md [hunk 4]
```diff
@@ -1,50 +1,46 @@
...
-    * On Windows, you might use "Visual Studio 12 2013" to generate a Visual Studio 2013 solution or "Visual Studio 14 2015 Win64" to generate a 64-bit Visual Studio 2015 solution.
-    * On OS X, you might use "Xcode" to generate an Xcode project
-    * On a UNIX-y system, simply omit the option to generate a makefile
+  * The `generator` option is the build system you'd like to use. Run `cmake` without arguments to see a full list of available generators.
+    * On Windows, you might use "Visual Studio 12 2013" (VS 2013 32-bits), or "Visual Studio 14 2015 Win64" (VS 2015 64-bits).
+    * On OS X, you might use "Xcode".
+    * On a UNIX-like system, omit the option (for a Makefile).
 
-  * yaml-cpp defaults to building a static library, but you may build a shared library by specifying `-DYAML_BUILD_SHARED_LIBS=ON`.
+  * `yaml-cpp` builds a static library by default, you may want to build a shared library by specifying `-DBUILD_SHARED_LIBS=ON`.
```
Comment (jbeder) [inline README.md]: YAML_BUILD_SHARED_LIBS

File: README.md [hunk 5]
```diff
@@ -1,50 +1,46 @@
...
+  * The command you'll need to run depends on the generator you chose earlier.
 
-5. To clean up, just remove the `build` directory.
+**Note:** To clean up, just remove the `build` directory.
 
-# Recent Release #
+## Recent Releases
 
-[yaml-cpp 0.6.0](https://github.com/jbeder/yaml-cpp/releases/tag/yaml-cpp-0.6.0) has been released! This release requires C++11, and no longer depends on Boost.
+[yaml-cpp 0.6.0](https://github.com/jbeder/yaml-cpp/releases/tag/yaml-cpp-0.6.0) released! This release requires C++11, and no longer depends on _Boost_.
```
Comment (jbeder) [inline README.md]: I personally wouldn't italicize Boost, but I guess not a big deal.

**Comment by KitsuneRal** [comment]
> 
> 
> Sounds good, a few small changes. I wish there were a way to previous the rendered markdown in github, but I guess that would be too user-friendly for them...

https://github.com/jbeder/yaml-cpp/pull/732/files?short_path=04c6e90#diff-04c6e90faac2675aa89e2176d2eec7d8 - is that what you're looking for? The respective button ("Display rich diff") is tucked on the toolbar for every given file (left of "Viewed" tickbox). Or you can display the resulting file without a diff (ellipsis to the right of the same tickbox).

**Comment by jbeder** [comment]
Ha, yes, thank you! Now I'll just grumble about how hard that button is to
find.

File: README.md (hunk 3 above)
Comment (marcospb19) [reply README.md]: This is the difference, the green arrow shows the topic with four "#"s

The text stands out

image

File: README.md (hunk 1 above)
Comment (marcospb19) [reply README.md]: Ok, `OFF` is on uppercase because it is the default.

File: README.md (hunk 5 above)
Comment (marcospb19) [reply README.md]: Removed

**Comment by marcospb19** [comment]
I checked `Allow edits from maintainers.` on this Pull Request, you can add any commits directly in this branch by editing README.md.

If you prefer this way, go to `Files Changed`, and then click on `Edit File`.

Then you can make the final changes and merge it.

**Comment by marcospb19** [comment]
hey @jbeder, is this going anywhere? It's been a long time, is there anything you want me to change?

I suggested you merging (if you liked the changes) and making a commit, or making a commit in my branch (which I allowed you to).

**Pull request merged**